crates/orderbook/src/orderbook.rs

```diff
@@ -42,24 +47,18 @@ struct Metrics {
     orders: prometheus::IntCounterVec,
 }
 
+#[derive(Display)]
+#[strum(serialize_all = "snake_case")]
 enum OrderOperation {
     Created,
```

Comment:
Looks like Created should take an `Option<Quote>` (since quotes are not relevant for cancelled operations). This would also make `on_order_operation` a bit cleaner (since right now there is an implicit assumption that quote is always None for `Cancellation` events)

Reply:
I will try to retrieve the quote for the cancelation events, so we can correctly label them.